setup/setup.sh

```diff
@@ -27,8 +27,7 @@ if [[ `grep "Kali GNU/Linux.*\(2\|Rolling\)" /etc/issue` ]]; then
 fi
 
 # make sure we run from this directory
-pushd . > /dev/null
-cd "$(dirname "$0")"
+pushd . > /dev/null && cd "$(dirname "$0")"
```

Comment:
Sorry, I might not have been clear enough in my previous comment. I meant ```pushd "$(dirname "$0")"```.

The current dir should already be on the stack
```
$ cd /
$ dirs -v
 0  /
$ pushd /etc > /dev/null
$ dirs -v
 0  /etc
 1  /
```